packages/react-ui/components/DateInput/__tests__/DateInput-test.tsx

```diff
@@ -236,4 +242,123 @@ describe('DateInput as InputlikeText', () => {
 
     expect(screen.getByTestId(InputLikeTextDataTids.nativeInput)).toBeDisabled();
   });
+
+  it('should handel keydown event', () => {
```

Comment:
```suggestion
  it('should handle keydown event', () => {
```

Хандель встречается ещё в двух тестах ниже :D

Reply:
Поправила)